### PR TITLE
Update WebFileRepository and XSOARWebServer integration files

### PR DESCRIPTION
**Pull request by ShirleyDenkberg**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/CRTXDOC-1811

## Description
Update parameters for EDL for XSOAR 8 and XSIAM

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/XsoarWebserver/Integrations/XSOARWebServer/XSOARWebServer.yml [hunk 1]
```diff
@@ -11,7 +11,8 @@ configuration:
   name: longRunning
   type: 8
   required: false
-- additionalinfo: The port on which the integration instance will listen
+- additionalinfo: Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use. (For Cortex XSOAR 8 and Cortex XSIAM) If you do not enter a Listen Port, an unused port for the EDL will automatically be generated when the instance is saved. However, if using an engine, you must enter a Listen Port.
```
Comment (ShahafBenYakir) [inline Packs/XsoarWebserver/Integrations/XSOARWebServer/XSOARWebServer.yml:14]:
```suggestion
- additionalinfo: Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use. (For Cortex XSOAR 8 and Cortex XSIAM) If you do not enter a Listen Port, an unused port for the XSOAR Web Server will automatically be generated when the instance is saved. However, if using an engine, you must enter a Listen Port.
```

File: Packs/WebFileRepository/Integrations/WebFileRepository/README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ This is helpful to make your environment ready for testing purpose for your play
     | --- | --- | --- |
     | Incident type |  | False |
     | Long running instance |  | False |
-    | Port mapping (&lt;port&gt; or &lt;host port&gt;:&lt;docker port&gt;) |  | True |
+    | Port mapping (&lt;port&gt; or &lt;host port&gt;:&lt;docker port&gt;) | Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use. (For Cortex XSOAR 8 and Cortex XSIAM) If you do not enter a Listen Port, an unused port for the EDL will automatically be generated when the instance is saved. However, if using an engine, you must enter a Listen Port.  | True |
```
Comment (ShirleyDenkberg) [inline Packs/WebFileRepository/Integrations/WebFileRepository/README.md:13]:
```suggestion
    | Port mapping (&lt;port&gt; or &lt;host port&gt;:&lt;docker port&gt;) | Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use. (For Cortex XSOAR 8 and Cortex XSIAM) If you do not enter a port, an unused port for Web File Repository will automatically be generated when the instance is saved. However, if using an engine, you must enter a port.  | True |
```

File: Packs/WebFileRepository/Integrations/WebFileRepository/README.md [hunk 3]
```diff
@@ -37,12 +37,14 @@ In a web browser, go to **`http://<cortex-xsoar-server-address>:<listen_port>`**
...
-2. In the **Server Configuration** section, verify that the `instance.execute.external.<instance_name>` key is set to `true`. If this key does not exist, click **+ Add Server Configuration** and add the `instance.execute.external.<instance_name>` and set the value to `true`. See [this documentation](https://xsoar.pan.dev/docs/reference/articles/long-running-invoke) for further information.
-3. In a web browser, go to `https://<cortex-xsoar-address>/instance/execute/<instance_name>/`.
+2. (For Cortex XSOAR 6.x) In the **Server Configuration** section, verify that the `instance.execute.external.<instance_name>` key is set to `true`. If this key does not exist, click **+ Add Server Configuration** and add the `instance.execute.external.<instance_name>` and set the value to `true`. See [this documentation](https://xsoar.pan.dev/docs/reference/articles/long-running-invoke) for further information.
+3. In a web browser, go to
+   (For Cortex XSOAR 6.x) `https://<cortex-xsoar-address>/instance/execute/<instance_name>/`.
 
    In Multi Tenant environments, go to `https://<cortex-xsoar-address>/acc_<account name>/instance/execute/<instance_name>/`
```
Comment (ShirleyDenkberg) [inline Packs/WebFileRepository/Integrations/WebFileRepository/README.md:44]:
```suggestion
3. In a web browser, 
   (For Cortex XSOAR 6.x) go to `https://<cortex-xsoar-address>/instance/execute/<instance_name>/`.

   In Multi Tenant environments, go to `https://<cortex-xsoar-address>/acc_<account name>/instance/execute/<instance_name>/`
```

File: Packs/WebFileRepository/Integrations/WebFileRepository/README.md [hunk 4]
```diff
@@ -37,12 +37,14 @@ In a web browser, go to **`http://<cortex-xsoar-server-address>:<listen_port>`**
...
 
-
+   For Cortex XSOAR 8 and Cortex XSIAM) `https://ext-<cortex-xsiam-address>/xsoar/instance/execute/<instance-name>`
```
Comment (ShirleyDenkberg) [inline Packs/WebFileRepository/Integrations/WebFileRepository/README.md:46]:
```suggestion
   (For Cortex XSOAR 8 and Cortex XSIAM) go to `https://ext-<cortex-xsiam-address>/xsoar/instance/execute/<instance-name>`
```

File: Packs/XsoarWebserver/Integrations/XSOARWebServer/README.md [hunk 5]
```diff
@@ -11,7 +11,8 @@ This integration was integrated and tested with version 1.0 of XSOAR-Web-Server
     | --- | --- | --- |
     | Incident type |  | False |
     | Long running instance |  | False |
-    | Server Listening Port | The port on which the integration instance will listen | True |
+    | Server Listening Port | Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use. (For Cortex XSOAR 8 and Cortex XSIAM) If you do not enter a Listen Port, an unused port for the EDL will automatically be generated when the instance is saved. However, if using an engine, you must enter a Listen Port.
```
Comment (ShirleyDenkberg) [inline Packs/XsoarWebserver/Integrations/XSOARWebServer/README.md:14]:
```suggestion
    | Server Listening Port | Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use. (For Cortex XSOAR 8 and Cortex XSIAM) If you do not enter a port, an unused port for XSOAR Web Server will automatically be generated when the instance is saved. However, if using an engine, you must enter a port.
```

**Comment by ShahafBenYakir** [comment]
@ShirleyDenkberg  I see that you have not fixed either yours or my comments.

File: Packs/WebFileRepository/Integrations/WebFileRepository/README.md [hunk 6]
```diff
@@ -37,12 +37,15 @@ In a web browser, go to **`http://<cortex-xsoar-server-address>:<listen_port>`**
...
+2. (For Cortex XSOAR 6.x) In the **Server Configuration** section, verify that the `instance.execute.external.<instance_name>` key is set to `true`. If this key does not exist, click **+ Add Server Configuration** and add the `instance.execute.external.<instance_name>` and set the value to `true`. See [this documentation](https://xsoar.pan.dev/docs/reference/articles/long-running-invoke) for further information.
+3. In a web browser, 
+   (For Cortex XSOAR 6.x) go to `https://<cortex-xsoar-address>/instance/execute/<instance_name>/`.
 
    In Multi Tenant environments, go to `https://<cortex-xsoar-address>/acc_<account name>/instance/execute/<instance_name>/`
 
-
+   (For Cortex XSOAR 8) go to `https://ext-<XSOAR-tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>`
+   (For Cortex XSIAM) go to `https://ext-<XSIAM-tenant>.crtx.<region>.paloaltonetworks.com/xsiam/instance/execute/<instance-name>`
+
```
Comment (ShahafBenYakir) [inline Packs/WebFileRepository/Integrations/WebFileRepository/README.md:48]:
```suggestion
      (For Cortex XSOAR 8 or XSAIM) go to `https://ext-<XSOAR-tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>`
```

**Review by ShahafBenYakir — APPROVED** [review]
One comment

**Pull request merged**